### PR TITLE
Update zh.po

### PR DESCRIPTION
**Pull request by moonmc666**

### CODE REVIEW

File: zh.po [hunk 1]
```diff
@@ -919,7 +919,7 @@ msgstr "删除图标"
 #, fuzzy, qt-format
```
Comment (peterix) [inline zh.po]: I you've changed it, it should no longer be marked as fuzzy

File: zh.po [hunk 2]
```diff
@@ -1026,7 +1026,7 @@ msgstr ""
 #, fuzzy
```
Comment (peterix) [inline zh.po]: Same

File: zh.po [hunk 3]
```diff
@@ -2413,13 +2414,13 @@ msgstr "支持MultiMC"
...
```
Comment (peterix) [inline zh.po]: Same

File: zh.po [hunk 4]
```diff
@@ -2413,13 +2414,13 @@ msgstr "支持MultiMC"
...
 msgctxt "MainWindow|"
 msgid "Minecraft Folder"
-msgstr "Minecraft日志"
+msgstr "游戏文件夹"
 
 #: src/application/MainWindow.cpp:525
 #, fuzzy
```
Comment (peterix) [inline zh.po]: Same

**Comment by peterix** [comment]
@cqjjjzr can you look at this please?

**Comment by cqjjjzr** [comment]
In summary: 
in this commit you:
1. changed all "you"(你) to the form of respect of "you" in Chinese("您")  
2. changed all uses of term "instance"(实例) to "game"(游戏)  
3. fixed very very few inarticulate expressions(caused by me)  
4. even not fixed some typo(父子->复制copy)  
5. added additional info to some English abbreviations or directly translate to Chinese (modpack->模组包, Zip->Zip压缩包(Zip compressed archive)), also some abbrs shouldn't be translated(e.g. mod loader)  
6. Label some label with "buggy", which is arrogating.(Start Minecraft maximized?->最大化运行Minecraft？(目前有BUG buggy at present))  
7. changed my name in the "Translators" field to his(WTF????????????)  

I really believe that this PR should entirely be not merged.

**Comment by cqjjjzr** [comment]
Also, I found some inarticulate expressions and typos written by me. I'll soon scan the translation and push a patch.

**Comment by cqjjjzr** [comment]
Since I pushed new commits to update and fix problems in zh.po, this pr will be closed.

**Comment by cqjjjzr** [comment]
Since the OP must be Chinese so I'll comment the changes in Chinese. @peterix let me know if you need a English ver.

1. 不行。根据@ustc_zzzz 的观点，结合在几个讨论群组中的调查：“您”本身就是一个不成熟的代替“你”的系统，请问“你们”怎么说？且“您”带有尊敬，谦让的情感色彩，作为一个工具，没有任何必要带有情感色彩，平铺直叙就是最好的；
2. 不行。实例就是实例，不是游戏。“游戏”这一名词在此处显得过于宽泛；
3. 我已经在最新的版本内修复了这些问题，见 9f88784ff03e89c694fe5051b48e2e9a414771dc。
4. 同上；
5. 反对将mod翻译为模组，modpack我已经在最新版本统一为“整合包”；
6. 反对。这是僭越行为，翻译中不应该含有超出原文意义的文本；
7. 我不想做评论。

**Pull request closed without merging**